### PR TITLE
Update api_server_audit_log_path to use different apiserver conf file.

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Change `api_server_audit_log_path` to check for `openshift-apiserver` api configuration file.
- Create a new rule to check the `openshift-kube-apiserver` namespace.
#### Rationale:

- Currently we are checking only the `openshift-kube-apiserver` but we also need to check the `openshift-apiserver` namespace for some rules.
- It looks like this PR is introducing something new and I don't know exactly we are going to handle these rules, if we are going to use the same group and such.

### CODE REVIEW

File: applications/openshift/api-server/api_server_audit_log_path/rule.yml [hunk 1]
```diff
@@ -45,9 +31,23 @@ ocil_clause: '<tt>audit-log-path</tt> does not contain a valid audit file path'
...
+    - general: |-
+        {{{ openshift_cluster_setting("/api/v1/namespaces/openshift-apiserver/configmaps/config") | indent(8) }}}
 {{%- endif %}}
-    The output should return a valid audit log path.
+
+template:
+    name: yamlfile_value
+    vars:
+        ocp_data: "true"
+        filepath: '/api/v1/namespaces/openshift-apiserver/configmaps/config'
```
Comment (JAORMX) [inline applications/openshift/api-server/api_server_audit_log_path/rule.yml:47]: I think this rule should check the kube apiserver.

File: ocp4/profiles/cis.profile [hunk 2]
```diff
@@ -60,6 +60,7 @@ selections:
     - api_server_profiling
   # 1.2.22 Ensure that the --audit-log-path argument is set
     - api_server_audit_log_path
+    - kube_api_server_audit_log_path
```
Comment (JAORMX) [inline ocp4/profiles/cis.profile:63]: I think we should instead explicitly create an extra rule that checks  for the openshift apiserver.

Something such as `openshift_apiserver_audit_log_path`. The rest of the rules that start with `api_server` check the kube-apiserver already.

**Review by JAORMX — CHANGES_REQUESTED** [review]
I'm still not sure how to handle rules that should apply to the openshift-apiserver. Maybe we could have a group for those? called `openshift-api-server`

File: ocp4/profiles/cis.profile (hunk 2 above)
Comment (ggbecker) [reply ocp4/profiles/cis.profile:63]: Yeah, I'm going to redo some stuff with these in mind. Thanks

File: applications/openshift/api-server/api_server_audit_log_path/rule.yml (hunk 1 above)
Comment (ggbecker) [reply applications/openshift/api-server/api_server_audit_log_path/rule.yml:47]: Got it. I will revert and improve how we are handling these rules.

**Pull request merged**